DemoApp-Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/HomeActivity.java

```diff
@@ -48,6 +52,17 @@ protected void onCreate(Bundle savedInstanceState)
         recyclerView.addItemDecoration( decoration );
         recyclerView.setItemAnimator( new DefaultItemAnimator() );
         recyclerView.setAdapter( adapter );
+
+        // Initialize the AppLovin SDK
+        AppLovinSdk.getInstance( this ).setMediationProvider( AppLovinMediationProvider.MAX );
+        AppLovinSdk.initializeSdk( this, new AppLovinSdk.SdkInitializationListener()
```

Comment:
Technically will match iOS implementations more, and keep more parity, if we do 

```suggestion
        AppLovinSdk.getInstance( this ).initializeSdk( this, new AppLovinSdk.SdkInitializationListener()
```